Dockerfile

```
@@ -0,0 +1,25 @@
+# = pgvector/postgres (ya trae psql y las contrib extensions: uuid-ossp, pg_trgm,
+#   fuzzystrmatch) + Flyway preinstalado (misma versión que usa esta action).
+#
+# Las migraciones NO se hornean acá: cada CI corre sus propias migraciones flyway
+# en build time según los parámetros que mande, así que no hay staleness. Esta
+# imagen solo elimina el costo por-run de instalar flyway + tooling de cliente y
+# de buildear el wrapper de postgresql-action.
+FROM pgvector/pgvector:0.8.2-pg16
+
+ENV FLYWAY_VERSION=8.5.11
+
+RUN set -eux; \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)